src/Slider.jsx

```diff
@@ -770,7 +765,6 @@ class Rheostat extends React.Component {
           ref={this.setHandleContainerNode}
           {...css(
             styles.handleContainer,
-            handleContainerStyle,
```

Comment:
Turns out that this didn't actually do anything, and was just being overridden. Since it is not overridden with the CSS interface, and was causing graphical issues, I removed it.